gradle/test-retry-gradle-plugin
# Fix build and enable pushing to remote build cache on CI

- Remove custom build cache config
- Enable pushing to remote build cache on CI
- Publish snapshots to repo.grdev.net
- Run sample against latest released version of Gradle
- Update to JUnit 4.13.2 to resolve security warning
- Stabilize Exemplar test
- Test against JUnit 5.8.2
- Update Gradle Enterprise plugins
- Make "Verify All" build a composite to aggregate information
